Kitodo-API/src/main/java/org/kitodo/api/dataeditor/rulesetmanagement/SimpleMetadataViewInterface.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.dataeditor.rulesetmanagement;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Provides an interface for the meta-data key view service. The meta-data key
+ * view service provides a filtered view on meta-data keys.
+ */
+public interface SimpleMetadataViewInterface extends MetadataViewInterface {
+    /**
+     * Maps a boolean value to a meta-data value.
+     *
+     * @param value
+     *            boolean input value
+     * @return value to save as a meta-data entry. If absent, delete the
+     *         meta-data entry.
+     */
+    default Optional<String> convertBoolean(boolean value) {
+        return value ? Optional.of(getSelectItems().entrySet().iterator().next().getKey()) : Optional.empty();
+    }
+
+    /**
+     * Returns the default value for the input type boolean.
+     *
+     * @return the default value
+     */
+    default boolean getBooleanDefaultValue() {
+        return !getDefaultItems().isEmpty();
+    }
+
+    /**
+     * Specifies the value for the entry that appears first when a field is
+     * added.
+     *
+     * @return the default value
+     */
+    Collection<String> getDefaultItems();
+
+    /**
+     * Returns the default value for all input types except boolean and multiple
+     * selection.
+     *
+     * @return the default value
+     */
```

Comment:
Why is there a list, if you always return the first object? And how can a value be valid for several input types?

Reply:
The rule set contains the element `<preset>` for the default values. However, the element can only occur once except for multiselect lists, so this method returns the first (and only) value. Multiselect lists, on the other hand, use the `getDefaultItems()` getter, which then returns all elements, because a multi-select list has more than one selected element (multiple `<preset>` elements in the rule set).

Reply:
_How can a value be valid for several input types?_ A value, not a specific value, but the fact that the number of default values is 1 may be correct for multiple input types. A simple selection list may have exactly one preselected value, a date may have exactly one value in advance, and so on.